test/suite/test_durable_rollback_to_stable.py

```diff
@@ -170,8 +169,8 @@ def test_durable_rollback_to_stable(self):
 
         # Use util to verify that second updates values have been flushed.
         errfilename = "verifyrollbackerr.out"
-        self.runWt(["verify", "-S", uri],
-            errfilename=errfilename, failure=True)
+        self.runWt(["verify", "-s", uri],
+            errfilename=errfilename, failure=False)
```

Comment:
We want to check that `wt` does NOT fail. Confusing flag since it's a double negative...